phpBB/phpbb/textformatter/s9e/factory.php

```diff
@@ -442,6 +442,20 @@ protected function configure_autolink(Configurator $configurator)
 	}
 
 	/**
+	* Escape a string to be used in an HTML attribute
+	*
+	* Escapes "HTML special chars" for obvious reasons and curly braces to avoid them
+	* being interpreted as an attribute value template
+	*
+	* @param  string $value Original string
+	* @return string        Escaped string
+	*/
+	protected function escape_html_attribute($value)
+	{
+		return htmlspecialchars(strtr($value, ['{' => '{{', '}' => '}}']));
```

Comment:
We could use `twig_escape_filter()` which implements a stronger escaper but I'm not sure it's a good idea....